posthog/session_recordings/models/session_recording.py

```diff
@@ -147,23 +147,14 @@ def check_viewed_for_user(self, user: Any, save_viewed=False) -> None:
             SessionRecordingViewed.objects.get_or_create(team=self.team, user=user, session_id=self.session_id)
             self.viewed = True
 
-    def build_object_storage_path(self, version: Literal["2023-08-01", "2022-12-22"]) -> str:
-        if version == "2022-12-22":
-            path_parts: list[str] = [
-                settings.OBJECT_STORAGE_SESSION_RECORDING_LTS_FOLDER,
-                f"team-{self.team_id}",
-                f"session-{self.session_id}",
-            ]
-            return "/".join(path_parts)
-        elif version == "2023-08-01":
-            return self._build_session_blob_path(settings.OBJECT_STORAGE_SESSION_RECORDING_LTS_FOLDER)
+    def build_object_lts_path(self, version: Literal["2023-08-01"]) -> str:
```

Comment:
this method name confused me even in the context of dealing with LTS storage -> rename